pepper-apis/config/application.conf.ctmpl

```diff
@@ -3,13 +3,13 @@
 {{with $environment := env "ENVIRONMENT"}}
 {{with $version := env "VERSION"}}
 {{with $app := env "APP"}}
-{{with $conf := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
-{{with $dsm_conf := vault (printf "secret/pepper/%s/%s/dsm/config" $environment $version)}}
+{{with $conf := secret (printf "secret/pepper/%s/%s/conf" $environment $version)}}
+{{with $dsm_conf := secret (printf "secret/pepper/%s/%s/dsm/config" $environment $version)}}
 {{with $auth0 := index $conf.Data "auth0"}}
-{{with $testingConf := vault (printf "secret/pepper/%s/%s/testing" $environment $version)}}
+{{with $testingConf := secret (printf "secret/pepper/%s/%s/testing" $environment $version)}}
 {{with $testingAuth := index $testingConf.Data "auth0"}}
 {{with $slack := index $conf.Data "slack"}}
-{{with $elasticsearchConf := vault (printf "secret/pepper/%s/%s/elasticsearch" $environment $version)}}
+{{with $elasticsearchConf := secret (printf "secret/pepper/%s/%s/elasticsearch" $environment $version)}}
 {
```

Comment:
All the template changes (.ctmpl files) are of the same kind: Newer consul-template versions (we were using a really old one) use the function `secret` instead of `vault`. 
Was forced to do this as only newer `consul-template` versions include an `arm64` build that will work on Apple M1s, so this is needed to keep `api-build.sh` working.